src/transformers/modeling_utils.py

```diff
@@ -460,6 +463,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
             resolved_archive_file = None
 
         # Instantiate model.
+        print(model_kwargs)
```

Comment:
Maybe remove this print?

Reply:
@LysandreJik Good catch! Totally missed it.